assets/i18n/en.json

```diff
@@ -66,8 +66,13 @@
     "I18N_ACTION_GUIDE_FOR_TEACHERS": "Guide for teachers",
     "I18N_ACTION_TIPS_FOR_PARENTS": "Tips for parents and guardians",
     "I18N_ACTION_VISIT_CLASSROOM": "Visit classroom",
+    "I18N_ADD_NEW_SYLLABUS_CANCEL_BUTTON_TEXT": "Cancel",
+    "I18N_ADD_NEW_SYLLABUS_DONE_BUTTON_TEXT": "Done",
+    "I18N_ADD_NEW_SYLLABUS_ITEMS": "New Syllabus Items",
     "I18N_ADD_SYLLABUS_DESCRIPTION_TEXT": "Add skills or stories to your syllabus to automatically send them to your students.",
     "I18N_ADD_SYLLABUS_SEARCH_PLACEHOLDER": "Search eg. Story,  Physics, English",
+    "I18N_ASSIGNED_STORIES_AND_SKILLS": "Assigned Stories and Skills",
+    "I18N_ASSIGNED_STORIES_AND_SKILLS_EMPTY_MESSAGE": "No stories or skills have been assigned to students of this group.",
```

Comment:
...students in this group...


Btw, I'm seeing the use of both "Learner" (e.g. Learner Group) and "Student" (in a bunch of the text). Which should it be? I thought we wanted to keep this consistent.

Reply:
Hmm, we are using "student" everywhere except for the feature name being called "Learner Group". There were a few inconsistencies which I have resolved.

Reply:
OK, have you checked/confirmed this with @dchen97 ? I'm fine with whatever she decides but I just want to make sure this has been consciously thought through.

Reply:
Hmm, I have sent in mail for confirming this. Can we get this PR with "students" being used consistently for now. Will create a separate PR for changing "students" to "learners" if required as there will be a lot of changes including variable names changes for making it consistent.

Reply:
I don't have strong objections, but we should get a decision on this (and a new issue filed, if necessary) before merging this in.

Reply:
Actually ... on second thoughts, better to just fix it in this PR IMO. Otherwise other people may start following the wrong convention you're introducing.

Try to ask for the decision to be made quickly, let them know it affects your PR merge schedule.